dahee/controllers/userController.js

```diff
@@ -17,6 +17,21 @@ const signUp = async (req, res) => {
   }
 };
 
+const signIn = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+    if (!email || !password) {
+      return res.status(400).json({ message: 'KEY_ERROR' });
+    }
+    const token = await userService.signIn(email, password);
+    res.status(200).json({ accessToken: token });
```

Comment:
가독성을 위해 구문들 사이에 공백을 추가해 주면 좋을 것 같습니다.
```suggestion
    const { email, password } = req.body;
    
    if (!email || !password) {
      return res.status(400).json({ message: 'KEY_ERROR' });
    }
    
    const token = await userService.signIn(email, password);
    res.status(200).json({ accessToken: token });
```